src/components/explore/QuantileExplorerView.svelte

```diff
@@ -194,20 +193,20 @@
                 comparisonKeyFormatter={(perc) => `${perc}%`}
                 yScaleType={probeType === 'log' ? 'scalePoint' : 'linear'}
                 {yDomain}
-                {smoothnessLevel}
+                {interpolate}
               >
                 <div slot="smoother" class="interpolator">
                   <input
                     id="toggleSmooth"
                     type="checkbox"
-                    bind:checked={smoothnessLevel}
+                    bind:checked={interpolate}
                   />
                   <h3 for="toggleSmooth" class="data-graphic__element-title">
-                    Interpolate
+                    Interpolated
                   </h3>
                   <span
                     use:tooltipAction={{
-                      text: 'Applies a moving average to smooth out short-term fluctuations on percentile values.',
+                      text: 'Generates percentiles using the Between Closest Ranks Linear Interpolation. This can show an innacurate representation of the data if the underlying distribution is not continuous and/or the data between bins is not uniformly distributed.',
```

Comment:
s/innacurate/inaccurate/

Also, this isn't quite right. This does not depend on the distribution of data *between* bins, only on the distribution of data *within* bins. It will actually handle any distribution between bins, which is fortunate since it's usually going to some weird multimodal exponential-ish thing.

Reply:
oops, I'll open another PR for this. I thought you were done.

Reply:
> oops, my bad. I'll open another PR for this. I thought you were done.

Sorry, I thought I was too.